setup.cfg

```diff
@@ -48,7 +48,7 @@ install_requires =
     invenio-oaiserver>=2.0.0,<3.0.0
     invenio-oauth2server>=2.0.0
     invenio-stats>=4.0.0,<5.0.0
-    invenio-vocabularies>=4.0.0,<5.0.0
+    invenio-vocabularies[s3fs]>=4.1.2,<5.0.0
```

Comment:
minor: what I had in mind was that end-users, in their Pipfile specify something like:

```ini
[packages]
...
# Note that we're not specifying a `version`, since we want this to come from
# our `invenio-app-rdm` dependency
invenio-vocabularies = {extras = ["s3fs", "oaipmh"]}
...
```